augly/image/transforms.py

```diff
@@ -188,6 +188,98 @@ def apply_transform(
         return F.apply_pil_filter(image, filter_type=self.filter_type, metadata=metadata)
 
 
+class DistortBarrel(BaseTransform):
+    def __init__(
+        self, a: float = 0.0, b: float = 0.0, c: float = 0.0, d: float = 1.0, p: float = 1
```

Comment:
supreme, teeny nit for consistency:
```suggestion
        self, a: float = 0.0, b: float = 0.0, c: float = 0.0, d: float = 1.0, p: float = 1.0
```